### ISSUE TITLE
Delayed wallet service start

### ISSUE THREAD

**Issue by drwasho** [OpenBazaar/openbazaar-go#1195]
Presently, the wallet service doesn't start until the IPFS part of OpenBazaar starts. It isn't clear to me why this is the case, but unfortunately the consequences of this are that a user has to wait at least a minute before they use the wallet, let alone accurately query what their last known balance was via the REST API.

This last point needs to be fixed **asap**.  We should be able to immediately query what the last known balance was as soon as the REST API is available rather than showing a balance of `0`. After the wallet service comes online, the balance can update as per normal. @cpacia @placer14 can we make this happen early this week?

**tl;dr**

- [ ] 1. The openbazaar-go should return the last known balance until the wallet service starts, rather than `0`
- [x] 2. The wallet service should start immediately if possible (rather than waiting for IPFS to start)

If nothing else, we need (1) **asap**.

---

Beyond that, since we'll be using an API based wallet by default, is there any reason to delay when the wallet service starts?

**Comment by rmisio** [comment]
> the consequences of this are that a user has to wait at least a minute before they use the wallet

Yikes, that wasn't an issue in v1. Isn't this supposed to be an improvement on the wallet?

What is on IPFS that's required in order to ping an insight server to get wallet balances?
